src/graph/types.ts

```diff
@@ -7,7 +7,11 @@ import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
  * In addition, this custom type also defines the optional `personImage` property,
  * which is used to pass the image around to other components as part of the person object.
  */
-export type IDynamicPerson = (MicrosoftGraph.User | MicrosoftGraph.Person | MicrosoftGraph.Contact) & {
+export type IDynamicPerson = (
+  | MicrosoftGraph.User
+  | MicrosoftGraph.Person
+  | MicrosoftGraph.Contact
+  | MicrosoftGraph.Group) & {
```

Comment:
Not a big fan of including groups as part of IdynamicPerson

Reply:
Reverted back to how it was :)